css/css-basics_pokemon-page/css/styles.css

```diff
@@ -4,12 +4,14 @@
 
 /* Task 1: First of all we want to adjust how the size of our elements is calculated when working with CSS. Do you know how to switch from "content-box" to "border-box"? */
 
-/* your solution */
+* {
+  box-sizing: border-box;
+}
 
 /* Task 2: Let's work on some basic stylings that we want to apply to all elements of your code via inheritance. Which selector do we need for this? */
 
 /* add selector */
- {
+body {
```

Comment:
If you put body here, the changes only affect elements inside the body. There is another solution to select all elements.

Reply:
you're right. I must use the `html` tag instead `body`